gitops-wg/README.MD

```diff
@@ -1,3 +1,89 @@
+
+## The What and Why of GitOps
+
+If you are new to GitOps, [it builds and iterates](https://www.weave.works/blog/gitops-operations-by-pull-request) on ideas drawn from DevOps and Infrastructure as Code that started with [Martin Fowler’s comprehensive Continuous Integration overview](https://martinfowler.com/articles/continuousIntegration.html) and provides the freedom to choose the tools that you need for your specific use cases.
+
+Individuals, teams, and organizations who implement GitOps experience many benefits, including:
+
+- Increased Developer & Operational Productivity
+- Enhanced Developer Experience
+- Improved Stability
+- Higher Reliability
+- Consistency and Standardization
+- Stronger Security Guarantees
+
+## GitOps Principles
+
+The group’s initial task – to "clearly define a vendor-neutral, principle-led meaning of GitOps" – is expressed as GitOps _Principles_, which are the foundation of GitOps practices.
+
+This aims to give participants in the cloud native ecosystem a common understanding of GitOps systems based on shared principles, rather than a matter of individual opinion or implementation preference.
+
+🚧 The GitOps Principles are currently a work in progress. For current status, follow [PR #48](https://github.com/gitops-working-group/gitops-working-group/pull/48).
+
+🕠 In the meantime, you may refer to the initial [Draft Definition](https://docs.google.com/document/d/11EZfvB2FFI837nMmArnyv-wizsIJvc-4_xdgfoUXF4o/view), which proposes the principles as follows:
+
+1. **Declarative Configuration**: All resources managed through a GitOps process must be completely expressed declaratively.
+2. **Version controlled, immutable storage**: Declarative descriptions are stored in a repository that supports immutability, versioning and version history. For example, git.
+3. **Automated delivery**: Delivery of the declarative descriptions, from the repository to runtime environment, is fully automated.
+4. **Software Agents**: Reconcilers maintain system state and apply the resources described in the declarative configuration.
+5. **Closed loop**: Actions are performed on divergence between the version controlled declarative configuration and the actual state of the target system.
+
+## How to Get Involved
+
+The GitOps Working Group is an open group, inviting companies and individuals to join and contribute to the community and the adoption of GitOps across the cloud native landscape.
+There are a few ways you can get involved:
+
+- Watch or star this repo to see when things change
+- Join the [GitOps Subreddit](https://www.reddit.com/r/GitOps/)
+- Attend a [Working Group or Committee meeting](#meetings)
+- [Open an issue](/../../issues/new) and let us know how you're using GitOps and any important considerations we should include
```

Comment:
This link must be updated to the issues of the `gitops-working-group` repo

Reply:
Ah yes, relative paths. Probably better now https://github.com/open-gitops/project.

Reply:
Resolved by omission. Created this tracking issue in OpenGitOps project: https://github.com/open-gitops/.github/issues/3